assisted-events-scrape/events_scrape/events_scrape.py

```diff
@@ -73,8 +73,10 @@ def run_service(self):
                             continue
                         else:
                             raise
-
-                self.process_cluster(cluster)
+                try:
+                    self.process_cluster(cluster)
+                except Exception:
+                    log.exception(f'Error while processing cluster {cluster_id}')
```

Comment:
can we write more details about the exception in the log?

Reply:
this should log the whole stacktrace, are you suggesting adding more context?
The idea is to keep it here while we add more specific exception handling according to errors we'll find